website/data/docs-remote-plugins.json

```diff
@@ -134,6 +134,13 @@
     "version": "latest",
     "pluginTier": "community"
   },
+  {
+    "title": "Vagrant",
+    "path": "vagrant",
+    "repo": "hashicorp/packer-plugin-vagrant",
+    "pluginTier": "official",
```

Comment:
Quick FYI. For `hashicorp` repos the default pluginTier is official if not provided. We are explicitly overriding for community plugins under `hashicorp`. If I'm repeating what you already know, please ignore.

Reply:
aaaaah I didn't know that, thanks.